api/external/infra_proxy/response/nodes.proto

```diff
@@ -13,16 +18,20 @@ message NodeAttribute {
     string id = 1;
     // Node name.
     string name = 2;
+    // Node FQDN.
+    string fqdn = 3;
+    // Node IP address.
+    string ip_address = 4;
     // Node last checkin.
-    string check_in = 3;
+    string check_in = 5;
     // Node uptime.
-    string uptime = 4;
+    string uptime = 6;
     // Node platform.
-    string platform = 5;
+    string platform = 7;
     // Node environment name.
-    string environment = 6;
+    string environment = 8;
     // Node policy group.
-    string policy_group = 7;
+    string policy_group = 9;
```

Comment:
Re-arranged it for better readability, as these APIs have not consumed yet.

Reply:
Reverted it back, proto gen validation won't approve these changes

Reply:
FYI: It is OK to make breaking changes when necessary, and certainly when the APIs are not yet in use. Just read the fine print on the error as to how to manage it:
```
🚨🚨🚨 If you NEED to get a breaking change through Ci: 🚨🚨🚨
add your file to the ignored files in buf.yaml and remove it in a subsequent pull request
```

Reply:
Gotch, we are good for now, this would be helpful thanks!